test/test_asr_quantize.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+# Copyright 2021 Gaopeng Xu
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+import pytest
+
+from espnet.nets.asr_interface import dynamic_import_asr
+
+torch = pytest.importorskip("torch")
```

Comment:
```suggestion
import torch
```
If you find this line in other modules, please ignore that. It is outdated.

Reply:
As CI failed, this test should be skipped if pytorch is too old for the feature. For example,
https://github.com/espnet/espnet/blob/ba564fc023765e1e0d6ebcb04efc1cb0da7b6205/test/test_recog.py#L19